src/components/settings/SettingsLMSTab/LMSCard.jsx

```diff
@@ -5,20 +5,21 @@ import { getLMSIcon } from '../../../utils';
 
 const cardText = 'Click "Configure" to get started';
 
-const LMSCard = ({ LMStype }) => (
+const LMSCard = ({ LMStype, onClick }) => (
   <Card style={{ width: '26rem' }}>
     <Card.Body>
       <h3>
         <Image className="lms-icon" src={getLMSIcon(LMStype)} />
         <span className="ml-2">{LMStype}</span>
       </h3>
       <p className="my-3">{cardText}</p>
-      <Button className="float-right" disabled>Configure</Button>
+      <Button onClick={() => onClick(LMStype)} className="float-right">Configure</Button>
```

Comment:
I'm so sorry for the confusion! Going along with the newest figma mocks, we're moving away from the button to config in favor of making the whole card clickable. Also, I believe the card text is being moved to header text on the whole section.
![image](https://user-images.githubusercontent.com/67655836/153274009-e112d2fd-70b8-4df6-8f0f-6657e3d08a4f.png)